plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/dpdk/DpdkDriverImpl.java

```diff
@@ -70,10 +70,12 @@ public void addDpdkPort(String bridgeName, String port, String vlan, DpdkHelper.
                 dpdkPortVhostUserClientType;
 
         StringBuilder stringBuilder = new StringBuilder();
-        stringBuilder.append(String.format("ovs-vsctl add-port %s %s " +
-                "vlan_mode=access tag=%s " +
-                "-- set Interface %s type=%s", bridgeName, port, vlan, port, type));
-
+        stringBuilder.append(String.format("ovs-vsctl add-port %s %s ", bridgeName, port));
+        if (Integer.parseInt(vlan) > 0 && Integer.parseInt(vlan) < 4095) {
+            stringBuilder.append(String.format("vlan_mode=access tag=%s ", vlan));
+        }
+        stringBuilder.append(String.format("-- set Interface %s type=%s", port, type));
+        
```

Comment:
Packages failing with: 
````
[ERROR] /jenkins/workspace/acs-pr-centos7-pkg-builder/dist/rpmbuild/BUILD/cloudstack-4.14.0.0-SNAPSHOT/plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/dpdk/DpdkDriverImpl.java:78: Line has trailing spaces. [RegexpSingleline]
````

@mbrashearnttglobalnet can you refactor this line space as a linux carriage?

Reply:
removed the white spaces on the blank line.